indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -129,14 +129,14 @@ private static Interval makeInterval(IndexIngestionSpec ingestionSchema, Granula
   }
 
   @JsonIgnore
-  private final IndexIngestionSpec ingestionSchema;
+  private final IndexIngestionSpec ingestionSpec;
 
   private final ObjectMapper jsonMapper;
 
   @JsonCreator
   public IndexTask(
       @JsonProperty("id") String id,
-      @JsonProperty("schema") IndexIngestionSpec ingestionSchema,
+      @JsonProperty("spec") IndexIngestionSpec ingestionSpec,
```

Comment:
no one uses this in production and i'm not aware of anyone using the new format for the index task